torch/csrc/distributed/rpc/utils.cpp

```diff
@@ -394,7 +394,7 @@ TensorPipeEntry tensorpipeSerialize(const Message& rpcMessage) {
     const auto& tensorDataVec = pickler.tensorData();
```

Comment:
@wanchaol would I be correct that these tensors would preserve its device affinity instead of always live on CPU? And this is because we no longer call `getWriteableTensorData` when pushing tensors into `tensor_data_` in JIT pickler?

Reply:
Yes that's right, the tensors will still alive on whatever device they were (no copy to cpu operations). tensorPipe serialize can use the tensors get from the pickler and pass around the data ptrs across GPUs.